metadata.yaml

```diff
@@ -15,3 +15,11 @@ maintainers:
 source: https://github.com/canonical/synapse-operator
 assumes:
   - k8s-api
+containers:
+  synapse:
+    resource: synapse-image
+resources:
+  synapse-image:
+    type: oci-image
+    description: Docker image for Synapse
+    auto-fetch: true
```

Comment:
I believe this is obsolete?

Reply:
Thanks! Since this PR has 2 approvals already, I'll merge it and then fix this on https://github.com/canonical/synapse-operator/pull/6 ok?